### PR TITLE
Check for valgrind power arch time base bug

### PR DESCRIPTION
**Pull request by nasastry**
This test verifies whether valgrind power architecture time base
bug exists or not.
This applies to 'ppc64le' architecture.

Signed-off-by: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by nasastry** [comment]
[fail_job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1757511/fail_job.log)
[pass_job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1757512/pass_job.log)

Attached Test fail job log and Test pass job log.

File: toolchain/power_time_base_bug.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    def setUp(self):
+        '''
+        Check for required packages namely gcc, make, valgrind.
+        Transfer the source file and make file.
+        Compile
+        '''
+        smm = SoftwareManager()
+        self.failures = []
+
+        dist = distro.detect()
```
Comment (harish-24) [inline toolchain/power_time_base_bug.py]: Looks to be unused. We can add a check to skip if `arch` is not `ppc64le`

File: toolchain/power_time_base_bug.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    def test(self):
+        '''
+        Execute the above compiled with and with out valgrind.
+        Expected output should be of 64-bit value
+        '''
+        os.chdir(self.teststmpdir)
+        cmd1 = "./print_power_time_base"
+        cmd2 = "valgrind ./print_power_time_base"
+        for cmd in cmd1, cmd2:
```
Comment (harish-24) [inline toolchain/power_time_base_bug.py]: Suggestion: `for cmd in ["./print_power_time_base", "valgrind ./print_power_time_base"]` to remove unnecessary variables.

File: toolchain/power_time_base_bug.py (hunk 1 above)
Comment (nasastry) [reply toolchain/power_time_base_bug.py]: @harish-24 Sure.

File: toolchain/power_time_base_bug.py (hunk 2 above)
Comment (nasastry) [reply toolchain/power_time_base_bug.py]: @harish-24 Sure.

**Comment by nasastry** [comment]
@harish-24 @PraveenPenguin Done with the required changes. Attaching the new fail, pass job logs.

[fail_job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1765977/fail_job.log)
[pass_job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1765978/pass_job.log)

**Comment by PraveenPenguin** [comment]
@harish-24 thanks for review

**Pull request merged**